tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml

```diff
@@ -399,7 +399,7 @@ jobs:
             1) docker run -v /usr/bin/qemu-aarch64-static:/usr/bin/qemu-aarch64-static -v $(Build.BinariesDirectory):/tmp/a -v $(Build.SourcesDirectory):/tmp/b -w /tmp/a --rm quay.io/pypa/manylinux2014_aarch64 /opt/python/'$(cp.tag)'/bin/python3 /tmp/b/setup.py bdist_wheel --nightly_build;;
             *) docker run -v /usr/bin/qemu-aarch64-static:/usr/bin/qemu-aarch64-static -v $(Build.BinariesDirectory):/tmp/a -v $(Build.SourcesDirectory):/tmp/b -w /tmp/a --rm quay.io/pypa/manylinux2014_aarch64 /opt/python/'$(cp.tag)'/bin/python3 /tmp/b/setup.py bdist_wheel;;
           esac
-          chown -R jclient:jclient .
+          sudo chown -R jclient:jclient .
```

Comment:
is sudo installed on the docker image? It is not a default package in images.

Another solution is to run `docker run -u0` and the image starts with root user. you can change accordingly later

Reply:
This is not runnning on docker.